net-wireless/unifi/files/unifi.initd

```
@@ -0,0 +1,27 @@
+#!/sbin/openrc-run
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+ACE_LIB="lib/ace.jar"
+DAEMON="${JAVA_HOME-/usr}"/bin/java
+JAVA_OPTS="-Dorg.xerial.snappy.tempdir=/var/lib/unifi/tmp"
+UNIFI_GROUP="unifi"
+UNIFI_PATH="/var/lib/unifi"
+UNIFI_USER="unifi"
+
+depend() {
+	need net
+}
+
+start() {
+	ebegin "Starting UniFi Controller"
+	cd "${UNIFI_PATH}"
```

`&&`, or this will fail silently.

---

Thanks, fixed.